sentry.client.config.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "",
+
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1,
+
+  // Setting this option to true will print useful information to the console while you're setting up Sentry.
+  debug: true,
+
+  replaysOnErrorSampleRate: 1.0,
+
+  // This sets the sample rate to be 10%. You may want this to be 100% while
+  // in development and sample at a lower rate in production
+  replaysSessionSampleRate: 0.1,
+
+  // You can remove this option if you're not planning to use the Sentry Session Replay feature:
+  integrations: [
+    new Sentry.Replay({
+      // Additional Replay configuration goes in here, for example:
+      maskAllText: true,
+      blockAllMedia: true,
+    }),
```

Comment:
Hmm, this would definitely be very useful, and [the privacy measures](https://docs.sentry.io/platforms/javascript/guides/nextjs/session-replay/privacy/) look pretty good, but I think I'd prefer to be one the safe side and not enable it at first, so we can get some more input on whether we actually should enable it.

Reply:
We need to be careful here, we don't want to accidentally collect PII. Why don't we gate this on `development` for now? Then we can't accidentally collect PII from production where it matters most.

Reply:
Oh right this is client-side, hm.

Reply:
OK using `NEXT_PUBLIC` variables seems to work fine here, thanks!

Reply:
Yes, dev only sounds good! And yeah, `NODE_ENV` is available everywhere just like `NEXT_PUBLIC_*` variables, IIRC.